### DISCUSSION TITLE
Request: Need North America Region / 433MHz for amateur radio licensees

### DISCUSSION THREAD

**Discussion by AlanTheBeast** [meshtastic/Discussions#56] [Firmware]
Title says it all, I believe.
Max power output should be allowed in this mode too.
Thanks.

**Comment by garthvh** [comment]
You can use Ukraine 433 for this

**Comment by AlanTheBeast** [comment]
Yes - entirely logical. /s

**Reply by garthvh** [reply]
80- 90% of meshtastic use is  in the US, if there was a US preset for the ham only band there would be a ton of misuse.

**Comment by AlanTheBeast** [comment]
I get the intention, thanks.

 But I think Europe is the busy spot - and in a denser area.  Where I am it’s barely known.

￼ 

￼

￼

**Reply by garthvh** [reply]
Germany is about 5% and the UK is about 5%, so tiny compared to the US

A higher percentage of European nodes are on mqtt maps, but usage is way lower.

**Comment by AlanTheBeast** [comment]
Consider Europe as “The United States of Europe”.

Not only more Meshtastic users (per liamcottle’s map, at least), but in a smaller geographical region.

About 3050 in US/Canada - very large area (discounting most of Canada of course)

About 4400 in western Europe - smaller area - more users.


Not to argue - but “as organized” now - it is not a clear thing.

**Comment by garthvh** [comment]
Still tiny in comparison, Canada is about half of Germany, the mqtt maps really over represent Europe.

**Comment by mc-hamster** [comment]
We have a workaround for this request -- that's to use an existing band within the range that compiles with the US ham band allocation. For the requestor, please use this. There is also the ability to use the frequency offset configuration if a wider selection is required.

This can be revisited if data indicates that there is enough usage that a dedicated band would be required. This won't be an easy fix as we would need to put in a few guard rails to ensure that this is not abused and that we maintain an acceptable user experience.
